README.md

```diff
@@ -8,22 +8,22 @@
 [![npm](https://img.shields.io/npm/v/@influxdata/influxdb-client)](https://www.npmjs.com/package/@influxdata/influxdb-client)
 [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://www.influxdata.com/slack)
 
-This repository contains the reference JavaScript client for InfluxDB 2.0. This client supports Node.js, browser, and Deno environments.
+This repository contains the reference JavaScript client for InfluxDB 2.x. This client supports Node.js, browser, and Deno environments.
 
 #### Note: Use this client library with InfluxDB 2.x. For connecting to InfluxDB 1.x see the [node-influx](https://github.com/node-influx/node-influx) client library. This library can also write/query [flux-enabled](https://docs.influxdata.com/influxdb/v1.8/administration/config/#flux-enabled--false) InfluxDB 1.8+.
 
 ## Documentation
 
 This section contains links to the client library documentation.
 
-- [Product documentation](https://docs.influxdata.com/influxdb/v2.0/api-guide/client-libraries/nodejs/), [Getting Started](#usage)
+- [Product documentation](https://docs.influxdata.com/influxdb/v2.1/api-guide/client-libraries/nodejs/), [Getting Started](#usage)
```

Comment:
Maybe a `latest` will be better:

```suggestion
- [Product documentation](https://docs.influxdata.com/influxdb/latest/api-guide/client-libraries/nodejs/), [Getting Started](#usage)
```

Reply:
`latest` is not precisely defined in terms of versioning, it might also point to `3.0` in the future that might come with some incompatibility (such as arrow results for queries). I would rather keep `2.1`.